### ISSUE TITLE
Please support sqlproj

### ISSUE THREAD

**Issue by ahdung** [lsoft/AdjustNamespace#3]
on sqlproj(sql server database project) not work.

**Comment by lsoft** [comment]
@ahdung if you are still interested in this, please compile this repo, install vsix, and give it a try. I've implemented few changes and it now works for me in my nano-sqlproj from `Tests`.

FYI: all `cs` files in `sqlproj` should have `Build Action = Compile` to be processed.

If everything is OK, I will publish a new version to VS masketplace.

**Comment by ahdung** [comment]
@lsoft EXCITING!! I might try it next tuesday.

**Comment by ahdung** [comment]
@lsoft I tried, not work, nothing happend after click.
![Recording 2022-11-21 at 23 33 48](https://user-images.githubusercontent.com/12094247/203094889-47fd339a-5525-45c1-92c4-160e60c2eaad.gif)

**Comment by lsoft** [comment]
@ahdung thanks. I'll take a look once again. Because I'm not familiar with sqlproj (at all), could you share your test project?

**Comment by lsoft** [comment]
@ahdung PTAL:

![Анимация](https://user-images.githubusercontent.com/5988558/203236823-3bf377eb-8a33-45c4-b076-babb54021276.gif)


there are a little chances to fix the issue without your sln, because everything works fine at my end. Also, 1) please confirm that you are using a version from repository (not from VS marketplace). 2) make note you need to build action set to "compile", otherwise Roslyn will not recognize your cs files.

**Comment by lsoft** [comment]
@ahdung I noticed your version is not recognizing your cs files, so 1) you are using version from VS marketplace 2) OR your files has not "Build action = Compile".

**Comment by ahdung** [comment]
@lsoft Sorry it's my fault, I do use the market one, so where is the repo one, link?

**Comment by lsoft** [comment]
@ahdung this repo. clone it, build, and take vsix file from `AdjustNamespace\AdjustNamespace.2022\bin\Debug\AdjustNamespace.vsix`


then uninstall your current version of AdjustNamespace and install the compiled one. Alternatively, you just can wait for a new version I publish, but I don't know when I'll do that.

**Comment by ahdung** [comment]
My VS does not install the plug-in development environment, and I don't want to install it, so can you please compile one for me?

**Comment by lsoft** [comment]
@ahdung I've published 0.1.11 to VS marketplace with the fix. You can update your Vsix as usual now.

**Comment by ahdung** [comment]
@lsoft Thank you very much! files in folder are ok, but files in root will be empty namespace, not apply `RootNamespace` property.
![Recording 2022-11-23 at 20 46 05](https://user-images.githubusercontent.com/12094247/203550882-1dcc6183-7395-4d50-b6f2-e99a861015fb.gif)

**Comment by ahdung** [comment]
Happy to this, I tried it, there are some issue I think.
1. The first part of namespace maybe should use `RootNamespace` property, not project name.
2. If a file not defined any namespace, this extension not apply it, by design? I hope provide a option.

**Comment by lsoft** [comment]
@lsoft 

> The first part of namespace maybe should use RootNamespace property, not project name.

good point, thanks. I fixed this via VS project system, due to the Roslyn project system does not provide such info for `sqlproj`. I've created a [bug](https://github.com/dotnet/roslyn/issues/65607), let's see.

Fixed in 0.1.13.

>If a file not defined any namespace, this extension not apply it, by design? 

the VSIX is adjusting existing namespaces, adding a new one is unexpected behaviour, I guess. So I prefer to leave this as is.
